root/etc/s6-overlay/s6-rc.d/init-mod-netbox-slurpit-install/run

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/with-contenv bash
+
+# This is an install script that is designed to run after init-mods-package-install
+# so it can take advantage of packages installed
+# init-mods-end depends on this script so that later init and services wait until this script exits
+
+echo "**** Installing Netbox SlurpIT plugin ****"
+echo "**** see https://netboxlabs.com/netbox-plugins/ for more information ****"
+
+if ! command -v python3; then
+    echo "**** Python3 is not installed ****"
+    exit 1
+else:
+    if [ -f /lsiopy/bin/python3 ]; then
+        /lsiopy/bin/python3 -m pip install --no-cache-dir slurpit_netbox
```

Comment:
We don't install packages in mods. Mods add the necessary packages into a file, to be installed by the package installer: https://github.com/linuxserver/docker-mods/blob/nextcloud-mediadc/root/etc/s6-overlay/s6-rc.d/init-mod-nextcloud-mediadc-add-package/run#L20

Reply:
I see that now. Is there a specific file that Python packages are written to, or do you just append them to the applications `requirements.txt`?

Reply:
you append any python package to `/mod-pip-packages-to-install.list` like in that link. And it has to run before the package installer